### PR TITLE
Make minor changes to setup.*

### PR DESCRIPTION
**Pull request by arcondello**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -10,6 +10,7 @@ classifiers =
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
     Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
```
Comment (thisac) [inline setup.cfg:13]: 👍

File: setup.py [hunk 2]
```diff
@@ -46,30 +46,20 @@ def build_extensions(self):
...
     name="dwave-gate",
     install_requires=[
         "numpy",
     ],
     ext_modules=cythonize(
-        extensions, annotate=bool(os.getenv("CYTHON_ANNOTATE", False))
+        ["dwave/gate/simulator/simulator.pyx",
+         "dwave/gate/registers/cyregister.pyx"
+         ],
+        annotate=bool(os.getenv("CYTHON_ANNOTATE", False))
```
Comment (thisac) [inline setup.py:58]: Does this just make it a bit more compact, or does it change anything else?

**Review by thisac — APPROVED** [review]

File: setup.py [hunk 3]
```diff
@@ -46,30 +46,20 @@ def build_extensions(self):
...
     ),
-    include_dirs=[numpy.get_include()],
+    include_dirs=[
+        numpy.get_include(),
+        'dwave/gate/simulator/'
+        ],
     cmdclass={"build_ext": build_ext_compiler_check},
-    zip_safe=False,
```
Comment (randomir) [inline setup.py:74]: This package is not `zip_safe`, but I just checked, `zip_safe` is now obsolete. I guess we stopped recommending `python install ..` route?

Comment (arcondello) [reply setup.py:74]: Oh, I removed it because it is specified in `setup.cfg`, not because it was obsolete

Comment (randomir) [reply setup.py:74]: Got it. Damned GiHub diffs. :laughing:

File: setup.py (hunk 2 above)
Comment (arcondello) [reply setup.py:58]: Mostly more compact

**Comment by arcondello** [comment]
I made another change to specify the include path in the file headers. That lets Cython have a better understanding of the relationship which is desirable when editing the files by hand

**Review by thisac — APPROVED** [review]
Looks good to me!

**Pull request merged**